app/views/hosts/_registration_tab.html.erb

```diff
@@ -0,0 +1,21 @@
+<script type="text/javascript">
+  function foreman_register_copy_configuration_to_clipboard(text) {
+    var aux = document.createElement("textarea");
+    $(aux).html(text);
+    document.body.appendChild(aux);
+    aux.select();
+    document.execCommand("copy");
+    document.body.removeChild(aux);
+  }
+</script>
+
+<% if @host.registration_template %>
+  <p>This host can be registered to Foreman by executing this command:</p>
```

Comment:
Could you expand on where you would see this view? I think it can be confusing to think of registering a host that already exists within Foreman and what all that means.

Reply:
I think the idea is that you create a host in Foreman and then it provides the instructions to actually change the host itself by installing/configuring the right tooling etc.